app/models/saved_claim/veteran_readiness_employment_claim.rb

```
@@ -294,14 +295,21 @@ def regional_office
     []
   end
 
-  def send_confirmation_email(user, service, _email_type, email_template)
+  # @see ::Logging::BenefitsIntake::Monitor#track_submission_exhaustion
+  def send_email(email_type)
+    VRE::NotificationEmail.new(id).deliver(email_type)
+  end
```

Take a look at [::Logging::BenefitsIntake::Monitor#track_submission_exhaustion](https://github.com/department-of-veterans-affairs/vets-api/blob/5ab59154aced747286ca188a89fbd1f36e7e1efa/lib/logging/benefits_intake/monitor.rb#L100-L109): this class must implement `send_email` (and have an email type of `:error`) in order to avoid silent failures.

---

Interesting. I looked at the code a bit and I see what you mean. Did you create the error template as part of this? I don't recall seeing it.